lib/reporting/drop_off_report.rb

```diff
@@ -399,6 +416,21 @@ def query
       QUERY
     end
 
+    def ipp_query
+      params = {
+        issuers: issuers.present? && quote(issuers),
+        event_names: quote(Results::IPP_ENROLLMENT_UPDATE),
+      }
+      format(<<~QUERY, params)
+        name
+        , properties.user_id AS user_id
+        #{issuers.present? ? '| filter properties.service_provider IN %{issuers}' : ''}
+        | filter name in %{event_names}
+        | filter properties.event_properties.passed or properties.event_properties.success
```

Comment:
1. do we need to say `passed = 1`? or `ispresent()`?
2. you can do this in one query, see the query we do for the identity verification report by checking event names

https://github.com/18F/identity-idp/blob/2f991d88cebbb4211342db5bbe2dae4d05d3e48e/lib/reporting/identity_verification_report.rb#L343-L344

here we could do

```
| filter (name in %{ipp_event_names} and properties.event_properties.passed = 1) or (name not in %{ipp_event_names})
```

Reply:
> here we could do
> 
> ```
> | filter (name in %{ipp_event_names} and properties.event_properties.passed = 1) or (name not in %{ipp_event_names})
> ```

This is great and I'll rewrite the query to use this. That said, more consistency in the properties of our events would also be good

Reply:
> do we need to say `passed = 1`? or `ispresent()`?

For the report data asked for, it's definitely `passed = 1`, `ispresent()` will return data we don't want

Reply:
> do we need to say `passed = 1`? or `ispresent()`?

We don't need to. Our code generating the event specifies it as `passed = true`. I'm happy to put it in as `passed = 1` if you think that's less ambiguous. We definitely get `passed = false` results included when querying with `ispresent()`